### ISSUE TITLE
[Feature] Add frontend for merging textures

### ISSUE THREAD

**Issue by 3vorp** [Faithful-Resource-Pack/App#124]
### Is your feature request related to a problem?

There's no good way to merge two textures currently without manually copying over everything.

### Describe the feature you'd like

The API backend for merging two textures has been done for some time (`PUT /textures/merge/{add_id}/{remove_id}`), but there's no frontend for actually doing it with a GUI. I'm not sure what the best way to do this would be — I'm thinking either a button on the texture modal next to the save button or a button in the texture search next to the edit/delete buttons could bring up a modal that lets you enter a texture ID to be merged in and deleted.

### Screenshot(s)

![image](https://github.com/Faithful-Resource-Pack/App/assets/75297863/6318d1d1-5704-484d-9589-1dcbc7101f15)
![image](https://github.com/Faithful-Resource-Pack/App/assets/75297863/feb01f50-a609-4d2b-918f-800c7f426dd0)

**Comment by TheRolfFR** [comment]
Don't merge this one without I give a personal touch to the code.
**Please** make a PR for it.

An icon button on the texture item is a better solution, because we wouldn't have to update the edit modal after the merge.

The texture to remove could be something more visual with a searchbox and an item preview.

The "texture to add" is terrible description, I would call it "Destination texture", and I would like the input to be disabled.

I would also add a confirmation sentence to avoid misclicks.

More generally, a little more visuals would help preview the changes.

**Comment by TheRolfFR** [comment]
Did you do something on this yet?

**Comment by 3vorp** [comment]
No, but I've also built the backend for the same feature for user merging and ID changing so whatever UI we come with here should probably be used for the user page.
